packages/space-opera/src/components/camera_settings/camera_settings.ts

```diff
@@ -66,23 +58,19 @@ class CameraOrbitEditor extends ConnectedLitElement {
   }
 
   render() {
-    if (!this.orbit)
-      return html``;
     return html`
       <div style="justify-content: space-between; width: 100%; display: flex;">
         <div>
           <me-draggable-input
             id="yaw"
             innerLabel="yaw"
-            value=${this.orbit.thetaDeg}
             min=-9999 max=9999
             style="min-width: 90px; max-width: 90px;"
             @change=${this.onChange}>
           </me-draggable-input>
           <me-draggable-input
             id="pitch"
             innerLabel="pitch"
-            value=${this.orbit.phiDeg}
```

Comment:
Now that you're removing these values. Will they be wrong when the model is initially loaded, and only be correct once you start dragging?

Reply:
Yeah, they aren't shown initially and I decided it quite a bit simpler to make them just sync one way. There are edge cases where you can update the snippet and it won't show in the UI, but I think this still overall has fewer bugs than where we were.